app/views/submissions/index.haml

```diff
@@ -30,7 +40,7 @@
             %td= submission.contributor_names
             %td= submission.status
 
--# reload page every 2s if submissions are still running
-- if @submissions.any?{ |s| !s.checks_completed? }
-  :javascript
-    setTimeout(function() { location.reload() }, 2000)
+-# -# reload page every 2s if submissions are still running
```

Comment:
cleaner to keep this here, and remove from line 16, unless you have a good reason to do so